File: app/src/test/java/medlife/com/br/model/CategoryTest.java
package medlife.com.br.model;

import static org.junit.Assert.*;
import org.junit.Before;
import org.junit.Test;

public class CategoryTest {
    private Category category;

    @Before
    public void setUp() {
        category = new Category(123, "Teste");
    }

    @Test
    public void testSetAndGetName() {
        category.setName("Medicamentos");
        assertEquals("Medicamentos", category.getName());
    }
} 

File: app/src/test/java/medlife/com/br/model/CategoryTest.kt
package medlife.com.br.model

import org.junit.Assert.assertEquals
import org.junit.Before
import org.junit.Test

class CategoryTest {
    private lateinit var category: Category

    @Before
    fun setUp() {
        category = Category(123, "Teste")
    }

    @Test
    fun testSetAndGetName() {
        category.name = "Medicamentos"
        assertEquals("Medicamentos", category.name)
    }
}
